### DISCUSSION TITLE
Lens navigation

### DISCUSSION THREAD

**Discussion by nrueckmann** [laravel/nova-issues#4174] [Ideas & Feature Requests] [answered]
Since the back button on lenses has been removed in Nova 4, i'm wondering how to navigate between lenses and/or their parent.
The current situation (using the main navigation to get back to resource) feels really wrong.

Are there any plans regarding UX for lenses?

**Comment by jonasbrenig** [comment]
Agreed, the back button should come back :)

Another thing to note is that now there is a full page reload that occurs when you are opening a lens from the index.

**Comment by crynobone** [comment, marked as answer]
You can enable navigation for Lens by enabling Breadcrumbs starting from 4.19.0

```php
Nova::withBreadcrumbs();
```
